agent/php_agent.c

```diff
@@ -87,6 +90,37 @@ zval* nr_php_get_zval_object_property(zval* object,
   return 0;
 }
 
+zval* nr_php_get_zval_base_exception_property(zval* exception,
+                                              const char* cname) {
+  char* name;
+  zend_class_entry* ce;
+
+  if ((NULL == exception) || (NULL == cname) || (0 == cname[0])) {
+    return 0;
+  }
+
+  name = (char*)nr_alloca(nr_strlen(cname) + 1);
+  nr_strcpy(name, cname);
```

Comment:
Why is this `strcpy` needed here?

Reply:
I am unsure. The copy is present in the sibling function that I based this one around, and I didn't want to risk changing it. I can remove the copy from both and see if it passes the tests...

Reply:
It still passes the CI, which I think is in line with our expectations. [e76d8e3](https://github.com/newrelic/newrelic-php-agent/pull/825/commits/e76d8e3b38a801ca6c407cabad0e13278dc1bfd1)